javascript/index.mjs

```diff
@@ -26,7 +26,7 @@
  * Class containing the diff, match and patch methods.
```

Comment:
The package is now declared `"type":"module"` so it should not be necessary to rename this file to `.mjs` extension now.

Reply:
not technically, but I figured it didn't hurt 🤔  happy to swap it back though